src/kex_code_mcbits/external/sk_gen.c

```diff
@@ -1,6 +1,6 @@
 #include "sk_gen.h"
 
-#include <nacl/randombytes.h>
+#include <randombytes.h>
```

Comment:
Sadly, with brew all the files are dumped directly in /usr/local/include, no nacl subdirectory. Maybe we'll have to do some kind of macro to distinguish the two cases?